microsoft/DiagManager
# Wave1 Fixes - Joe

Fixes # .

Changes proposed in this pull request:
- Minor bug fixes

 

How to test this code:
 - Run pssdiag.ps1 and checkout the command line and its arguments
 - Also see the output of tempdb_and_tarn output
